syncapi/sync/requestpool.go

```diff
@@ -132,10 +133,24 @@ func (rp *RequestPool) OnIncomingSyncRequest(req *http.Request, device *authtype
 
 func (rp *RequestPool) currentSyncForUser(req syncRequest, latestPos types.PaginationToken) (res *types.Response, err error) {
 	// TODO: handle ignored users
```

Comment:
If you're implementing handling of ignored users then we probably want this comment removed.